src/web/services/userAccount.ts

```diff
@@ -74,7 +74,6 @@ export async function GetAllUsersOfParticipant(participantId: number) {
   }
 }
 
-// TODO: make this only remove the user from the given participant in UID2-3852
```

Comment:
nit: should you change the error message below to 'Could not remove user' instead of delete

Reply:
done!